src/nested_pandas/nestedframe/core.py

```diff
@@ -777,6 +777,76 @@ def min(self, exclude_nest: bool = False, numeric_only: bool = False, **kwargs):
         else:
             return pd.concat([base_min] + nested_mins)
 
+    def max(self, exclude_nest: bool = False, numeric_only: bool = False, **kwargs):
+        """
+
+        Return the maximum value of each column as a series, including nested columns
+        with prefix to indicate the source column.
+
+        This computes the column-wise maximum (axis=0) across base and nested columns.
+        Row-wise maximum (axis=1) are not supported, as reductions along columns
+        are the primary intended behavior for NestedFrame.
+
+        By default, missing values (NaNs) will be skipped in the computation.
+
+        For non-numeric columns (e.g., strings), the method returns the
+        lexicographically largest value when `numeric_only=False` (default).
+
+        Parameters
+        ----------
+        exclude_nest : bool, default False
+            If set to True, will exclude the nested structure and
+            only computes the maximum over the base columns
+        numeric_only : bool, default False
+            Include only float, int, boolean columns.
+        **kwargs
+            See the documentation for :meth:`pandas.DataFrame.max`
+            for complete details on the keyword arguments accepted by
+            :meth:`max`.
+
+        Returns
+        -------
+        pd.Series
```

Comment:
```suggestion
        pandas.Series
```

Intersphinx!  Without the full package name, the link won't render.  Check your `min` docstring too.